File: app/src/main/java/com/futhark/android/seevoice/model/domain/SeeVoiceAccount.java
package com.futhark.android.seevoice.model.domain;

import android.accounts.Account;
import android.os.Parcel;

import java.sql.Date;

/**
 * Created by liuhr on 14/04/2017.
 */

public class SeeVoiceAccount extends Account {
    public SeeVoiceAccount(String name, String type) {
        super(name, type);
    }

    public SeeVoiceAccount(Parcel in) {
        super(in);
    }

    private String domain;
    private String type;
    private Date createAt;
}

File: app/src/main/java/com/futhark/android/seevoice/model/domain/SeeVoiceAccount.kt
package com.futhark.android.seevoice.model.domain

import android.accounts.Account
import android.os.Parcel

import java.sql.Date

/**
 * Created by liuhr on 14/04/2017.
 */

class SeeVoiceAccount : Account {

    private val domain: String? = null
    private val type: String? = null
    private val createAt: Date? = null

    constructor(name: String, type: String) : super(name, type) {}

    constructor(`in`: Parcel) : super(`in`) {}
}
